src/utils/index.js

```diff
@@ -40,11 +40,17 @@ export function getCookie(name) {
   return value !== null ? decodeURIComponent(value[1]) : null;
 }
 
-export function getPastTime(time) {
+export function getPastTime(time, locale = 'zh') {
   const now = new Date();
   const date = new Date(time);
   const diff = (now.getTime() - date.getTime()) / (60 * 60 * 1000);
-  return diff / 24 > 1 ? parseInt(diff / 24) + ' days ago' : parseInt(diff) + ' hours ago';
+  let daysago = ' 天前',
```

Comment:
I think put locale string in this function is not good. What if there are other locales ?

Reply:
ok, add a translation.js  for js file word translation.